redfish-core/lib/service_root.hpp

```diff
@@ -23,6 +23,105 @@
+                                    const boost::system::error_code ec2,
+                                    const std::vector<
+                                        std::pair<std::string, VariantType>>&
+                                        propertiesList) {
+                                    if (ec2)
+                                    {
+                                        // doesn't have to include this
+                                        // interface
+                                        return;
+                                    }
+                                    BMCWEB_LOG_DEBUG
+                                        << "Got " << propertiesList.size()
+                                        << " properties for system";
+                                    for (const std::pair<std::string,
+                                                         VariantType>&
+                                             property : propertiesList)
+                                    {
+                                        const std::string& propertyName =
+                                            property.first;
+                                        if ((propertyName == "Model") ||
+                                            (propertyName == "SerialNumber"))
+                                        {
+                                            const std::string* value =
+                                                std::get_if<std::string>(
+                                                    &property.second);
+                                            if (value != nullptr)
+                                            {
+                                                asyncResp->res
+                                                    .jsonValue["ACF"]
+                                                              [propertyName] =
+                                                    *value;
+                                            }
+                                        }
+                                    }
+                                },
+                                connection.first, path,
+                                "org.freedesktop.DBus.Properties", "GetAll",
+                                "xyz.openbmc_project.Inventory.Decorator."
+                                "Asset");
+                        }
+                    }
+                }
+            }
+        },
+        "xyz.openbmc_project.ObjectMapper",
+        "/xyz/openbmc_project/object_mapper",
+        "xyz.openbmc_project.ObjectMapper", "GetSubTree",
+        "/xyz/openbmc_project/inventory", int32_t(0),
+        std::array<const char*, 1>{
+            "xyz.openbmc_project.Inventory.Decorator.Asset",
+        });
+
+    std::pair<std::string, std::string> redfishDateTimeOffset =
+        crow::utility::getDateTimeOffsetNow();
+
+    asyncResp->res.jsonValue["ACF"]["DateTime"] = redfishDateTimeOffset.first;
```

Comment:
I would think this should be ["Oem"]["OpenBMC"]["DateTime"] or ["Oem"]["IBM"]["DateTime"]

Reply:
The other thing you could do is ["Oem"]["OpenBMC"]["ACF"]["DateTime"]
which is what 
https://github.com/ibm-openbmc/bmcweb/blob/1020/static/redfish/v1/schema/OemManagerAccount.v1_0_0.xml
does.. but i don't think knowing this is for ACF helps us

Reply:
Done.